ui/imports/shared/views/HistoryView.qml

```diff
@@ -116,20 +94,53 @@ ColumnLayout {
             property bool modelDataValid: modelData !== undefined && !!modelData
             isIncoming: modelDataValid ? modelData.to === account.address: false
             cryptoValue: modelDataValid ? modelData.value : undefined
-            fiatValue: modelDataValid ? RootStore.getFiatValue(cryptoValue.amount, symbol, RootStore.currentCurrency) : undefined
+            fiatValue: modelDataValid  && cryptoValue !== undefined && !!cryptoValue ? RootStore.getFiatValue(cryptoValue.amount, symbol, RootStore.currentCurrency) : undefined
```

Comment:
`cryptoValue !== undefined && !!cryptoValue`

Isn't that the same effectively?

Reply:
Yep I just tried checking and it is effectively one and the same will remove